### DISCUSSION TITLE
How lenis works ?

### DISCUSSION THREAD

**Discussion by TheElegantCoding** [darkroomengineering/lenis#364] [Q&A] [answered]
I have a question about how this package works, this make a wrapper with position fixed for all the page and add a transition when user scroll ? Or this modify the event of scroll in js

**Comment by clementroche** [comment, marked as answer]
Lenis prevent the default `wheel` event and animate the scroll according to the `wheel.deltaY` value, it doesn't require any `position: fixed`.
